logging/bin/deploy_fluentbit_azmonitor.sh

```diff
@@ -1,46 +1,55 @@
-#! /bin/bash
+#!/bin/bash
 
 # Copyright © 2020, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-cd "$(dirname $BASH_SOURCE)/../.."
+# shellcheck source=/dev/null
```

Comment:
What's this line do?

Reply:
it was an original ref to make spellcheck ignore the source but I ignored it globally. I removed comment.